index.js

```diff
@@ -1,99 +1,111 @@
 // express.js is server side and only runs in node - not a browser. This excludes loading it for the web.
 try {
-    var express = require('express');
```

Comment:
Sorry for the mismatch on tab spacing, will avoid doing this in the future so the diffs are clearer